### ISSUE TITLE
Rotation lock

### ISSUE THREAD

**Issue by Octcent** [kolyvan/kybook#69]
Hope can adjust rotation lock anytime

**Comment by karlic** [comment]
Your issue is very vague. Can you add more detail? Use Google Translate if that helps. 
https://translate.google.co.uk/m/translate

Rotation lock is only available in KyBook 3 from version 0.5.5 (http://kybook-reader.com/changelog.html). You can also use the device’s rotation lock. Swipe up from the bottom of the screen to reveal Apple’s Control Center. Locate the round lock symbol button. Tap this and the button will turn white indicating that the screen orientation is now locked. The same lock symbol will then appear in the top status bar to help remind you.

**Comment by Octcent** [comment]
I don't want to apply system orientation. And it looks KyBook 3 require iOS 11+. I don't know if the feature is what I want.

I hope there is an option to fix the rotation of the whole app, and this option is not related to "while reading"

**Comment by karlic** [comment]
> I don't want to apply system orientation. I hope there is an option to fix the rotation ...

It has already been added to the iPhone. The other reading apps I have tried have rotation lock (otherwise the book spins round every time you move!), so I can't believe it won't get added to the iPad. (It also makes sense in KyBook 3 for the scroll lock feature , https://github.com/kolyvan/kybook/issues/59)

> And it looks KyBook 3 require iOS 11+

I think so. It doesn't install on my iPhone (iOS 9.3.5)
Confirmed here: http://kybook-reader.com

> ...  of the whole app, and this option is not related to "while reading"

So you want to lock the whole app, not just when a book is displayed?  Why is that?
